app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -799,42 +799,43 @@ class PatientSummaryScreen :
   }
 
   override fun showStatinAlert(statin: StatinModel) {
-    val patientText = getString(R.string.statin_alert_patient)
-    var statinAlertDescriptionText = ""
-
-    if (statin.hasDiabetes) {
-      statinAlertDescriptionText = String.format(getString(R.string.statin_alert_has_diabetes), statin.age.toString())
+    val statinAlertDescriptionText = buildString {
+      append("${getString(R.string.statin_alert_patient)} ")
+      if (statin.hasDiabetes) {
+        append(String.format(getString(R.string.statin_alert_has_diabetes), statin.age.toString()))
+      }
+      if (statin.hasHadStroke || statin.hasHadHeartAttack) {
+        append(getCVHHistoryString(statin))
+      }
+      append(".")
     }
 
+    statinAlertDescription.text = statinAlertDescriptionText
+    statinAlertView.visibility = VISIBLE
+  }
+
+  private fun getCVHHistoryString(statin: StatinModel): String {
     val historyList = mutableListOf<String>()
     if (statin.hasHadStroke) {
       historyList.add(getString(R.string.statin_alert_stroke))
     }
     if (statin.hasHadHeartAttack) {
       historyList.add(getString(R.string.statin_alert_heart_attack))
     }
-
-    if (historyList.isNotEmpty()) {
-      val historyText = when (historyList.size) {
-        1 -> if (statin.hasDiabetes) {
-          " ${getString(R.string.statin_alert_and_seperator)} ${getString(R.string.statin_alert_cvh_history)} ${historyList[0]}"
-        } else {
-          "${getString(R.string.statin_alert_cvh_history)} ${historyList[0]}"
-        }
-
-        else -> if (statin.hasDiabetes) {
-          ", ${getString(R.string.statin_alert_cvh_history)} ${historyList.joinToString(" ${getString(R.string.statin_alert_and_seperator)} ")}"
-        } else {
-          "${getString(R.string.statin_alert_cvh_history)} ${historyList.joinToString(" ${getString(R.string.statin_alert_and_seperator)} ")}"
-        }
+    val historyText = when (historyList.size) {
+      1 -> if (statin.hasDiabetes) {
+        " ${getString(R.string.statin_alert_and_seperator)} ${getString(R.string.statin_alert_cvh_history)} ${historyList[0]}"
+      } else {
+        "${getString(R.string.statin_alert_cvh_history)} ${historyList[0]}"
       }
 
-      statinAlertDescriptionText += historyText
+      else -> if (statin.hasDiabetes) {
+        ", ${getString(R.string.statin_alert_cvh_history)} ${historyList.joinToString(" ${getString(R.string.statin_alert_and_seperator)} ")}"
+      } else {
+        "${getString(R.string.statin_alert_cvh_history)} ${historyList.joinToString(" ${getString(R.string.statin_alert_and_seperator)} ")}"
+      }
```

Comment:
Just for my visual understanding, can you mention what the final strings are going to look like for these conditionals?

With 1 medical history or both medical histories.

Reply:
You can also just link to it

Reply:
When the patient is diabetic and has no history of cvh history: `Patient is 48 with diabetes.`
When the patient is diabetic and has 1 medical history: `Patient is 48 with diabetes and has history of heart attack.`
When the patient is diabetic and has both medical history: `Patient is 48 with diabetes and has history of heart attack and stroke.`
When the patient is non-diabetic and has 1 medical history: `Patient has history of heart attack.`
When the patient is diabetic and has both medical history: `Patient has history of heart attack and stroke.`

Reply:
https://www.figma.com/design/vmZlmlFSCOrEYzTMkqsNB8/Simple-(Production)?node-id=32039-4753&t=vDCkQzAcBHopsxkJ-4